### PR TITLE
PTT print related fixes

### PR DESCRIPTION
**Pull request by rendesg**
Hi, I just found your library, and found very handy! :)

I happened to found some minor problems which prevent me to make PTT-print work, could you please review & merge these updates?

Thank you!

### CODE REVIEW

File: android/src/main/kotlin/com/rouninlabs/another_brother/AnotherBrotherPlugin.kt [hunk 1]
```diff
@@ -125,6 +125,9 @@ class AnotherBrotherPlugin : FlutterPlugin, MethodCallHandler {
         }
         else if (call.method == ReplaceTextNameMethodCall.METHOD_NAME) {
             ReplaceTextNameMethodCall(flutterAssets = mFlutterAssets, context = mContext, call = call, result = result).execute()
+        }
+		else if (call.method == FlushPttPrintMethodCall.METHOD_NAME) {
```
Comment (CodeMinion) [inline android/src/main/kotlin/com/rouninlabs/another_brother/AnotherBrotherPlugin.kt:129]: Thanks for the update @rendesg. Looks great, if you wouldn't mind could you fix up the formatting here and we'll go ahead and merge this to main. 

Thanks again!

File: lib/printer_info.dart [hunk 2]
```diff
@@ -3100,7 +3100,7 @@ class Printer {
       "printerId": mPrinterId,
       "printInfo": mPrinterInfo.toMap(),
       "key": key,
-      "encode":key
```
Comment (CodeMinion) [inline lib/printer_info.dart:3103]: Great catch @rendesg! Thanks for the PR.

**Review by CodeMinion — CHANGES_REQUESTED** [review]
This looks great @rendesg ! Just added a minor comment on the formatting in one of the classes so once that's I we can go ahead and merge. 

I can also fix the formatting after merging, but if we can do it here it might be quicker. Thanks!

**Comment by rendesg** [comment]
Sorry, I was using some plain text editor so I have accidentally mixed up the formatting and it looked fine for me (tabs vs spaces).
I hope it's fine now!

File: android/src/main/kotlin/com/rouninlabs/another_brother/AnotherBrotherPlugin.kt (hunk 1 above)
Comment (rendesg) [reply android/src/main/kotlin/com/rouninlabs/another_brother/AnotherBrotherPlugin.kt]: Sorry, I was using some plain text editor so I have accidentally mixed up the formatting and it looked fine for me (tabs vs spaces).
I hope it's fine now!

File: lib/printer_info.dart (hunk 2 above)
Comment (rendesg) [reply lib/printer_info.dart:3103]: You're welcome, thank you for the library ;)

**Comment by rendesg** [comment]
Sorry, I was using some plain text editor so I have accidentally mixed up the formatting and it looked fine for me (tabs vs spaces).
I hope it's fine now!

**Review by CodeMinion — APPROVED** [review]
Awesome! Thanks a lot for the fix @rendesg

**Comment by rendesg** [comment]
@CodeMinion : could you please prepare/release a new version including these fixes? 
That would really help us out, thank you!

**Comment by CodeMinion** [comment]
Thanks for the reminder @rendesg version 0.0.23 is now available at pub.dev. 
I also began adding apps built with another_brother to the lib so if you would like your app listed just send me a link and I'll add it. 

Thank again

**Pull request merged**